apps/evm/src/pages/Market/Market/MarketHistory/Card/CapThreshold/index.tsx

```diff
@@ -0,0 +1,151 @@
+      value: capTokens,
+      token,
+      addSymbol: false,
+    });
+
+    const tmpReadableDeltaDollars = formatCentsToReadableValue({
+      value: tmpCapCents.minus(tmpBalanceCents),
+    });
+
+    const tmpReadableDeltaTokens = formatTokensToReadableValue({
+      value: capTokens.isEqualTo(0) ? new BigNumber(0) : capTokens.minus(balanceTokens),
+      token,
+    });
+
+    const thresholdPercentage = capTokens.isEqualTo(0)
+      ? 100
+      : balanceTokens.multipliedBy(100).div(capTokens).toNumber();
+
+    const tmpReadableThresholdPercentage = formatPercentageToReadableValue(thresholdPercentage);
+
+    return {
+      readableBalanceDollars: tmpReadableBalanceDollars,
+      readableBalanceTokens: tmpReadableBalanceTokens,
+      readableCapDollars: tmpReadableCapDollars,
+      readableCapTokens: tmpReadableCapTokens,
+      readableDeltaDollars: tmpReadableDeltaDollars,
+      readableDeltaTokens: tmpReadableDeltaTokens,
+      readableThresholdPercentage: tmpReadableThresholdPercentage,
+      thresholdPercentage,
+    };
+  }, [balanceTokens, capTokens, tokenPriceCents, token]);
+
+  return (
+    <div className="flex items-center space-x-4">
+      <Tooltip
+        title={
+          type === 'supply'
+            ? t('market.supplyCapThreshold.tooltip', {
+                amountDollars: readableDeltaDollars,
+                amountTokens: readableDeltaTokens,
+              })
+            : t('market.borrowCapThreshold.tooltip', {
+                amountDollars: readableDeltaDollars,
+                amountTokens: readableDeltaTokens,
+              })
+        }
+      >
+        <div className="relative flex items-center justify-center w-20 h-20">
+          <ProgressCircle
+            value={thresholdPercentage}
+            sizePx={80}
+            strokeWidthPx={5}
+            className="absolute inset"
+            fillColor={`url(#${THRESHOLD_GRADIENT_ID})`}
+            defs={
+              <linearGradient
+                id={THRESHOLD_GRADIENT_ID}
+                x1="68.5998"
+                y1="55.4944"
+                x2="-13.4919"
+                y2="6.11727"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stop-color={theme.colors.blue} />
+                <stop offset="1" stop-color="#5CFFA2" />
+              </linearGradient>
+            }
+          />
+
+          <p className="text-sm text-center font-bold">{readableThresholdPercentage}</p>
+        </div>
+      </Tooltip>
+
+      <div>
+        <p className="text-grey mb-1 text-sm">
+          {type === 'supply'
+            ? t('market.supplyCapThreshold.title')
+            : t('market.borrowCapThreshold.title')}
+        </p>
+
+        <p className="text-sm font-semibold sm:text-lg">
+          {readableBalanceDollars} / {readableCapDollars}
+        </p>
+
+        <p className="text-grey text-xs">
+          {readableBalanceTokens} / {readableCapTokens}
```

Comment:
Shouldn't we add the token symbol here? I feel the numbers alone don't say what they mean, they end looking like this: 0.052792 / 10K

Reply:
Sure thing! And one additional question here: should this string be moved to the en.json file and then accessed through the translation layer?

Reply:
In this I don't think so, because it would always be displayed the same regardless of the language used on the UI.